python/py-flup/Portfile

```diff
@@ -5,7 +5,8 @@ PortGroup           python 1.0
 
 name                py-flup
 license             BSD
-version             1.0.3.dev-20110405
+version             1.0.3
+revision            1
```

Comment:
If you are trying to trigger an update because the version "1.0.3" is alphabetically less than "1.0.3.dev-20110405", then incrementing the `revision` won't work. You have to instead add the epoch line:

`epoch    1
`

Reply:
thanks Frank, that was indeed my intention... fixed another mistake in the same port, the "final" 1.0.3 version doesn't have any files in the archive anymore to install in post-destroot. This should fix the CI error as well.